sqlglot/tokens.py

```diff
@@ -414,6 +414,7 @@ class TokenType(AutoName):
     SOURCE = auto()
     ANALYZE = auto()
     NAMESPACE = auto()
+    EXPORT = auto()
```

Comment:
This needs to go in `ID_VAR_TOKENS`, otherwise `select 1 export` will break despite being valid.